delivery/services/staging_service.py

```diff
@@ -104,8 +102,8 @@ def _copy_dir(staging_order_id, external_program_service, session_factory, stagi
                 # Parse the file size from the output of rsync stats:
                 # Total file size: 207,707,566 bytes
                 match = re.search(r'Total file size: ([\d,]+) bytes',
-                                  execution_result.stdout,
-                                  re.MULTILINE)
+                                 execution_result.stdout,
+                                 re.MULTILINE)
```

Comment:
This was correctly indented before :( lines 105 and 106 should be aligned with the first parameter of the search function above in this case.